application/core/main.go

```diff
@@ -330,6 +408,7 @@ func main() {
 	protected.GET("/tags", getTags)
 	protected.GET("/matchings-stats", handler.GetMatchingStats)
 	protected.GET("/links", getLinks)
+	protected.GET("/execute-schedule/:organization", scheduleJob)
```

Comment:
this endpoint is good and is handling all scheduled of a given organization.
How should we handle it for all the organizations? I believe we need one thing for that because the script will not guess or try to call functions to get all the organizations